Makefile

```diff
@@ -795,11 +795,10 @@ pkgs: build-tools $(PKGS)
 
 pkg/external-boot-image/build.yml: pkg/external-boot-image/build.yml.in
 	$(QUIET)tools/compose-external-boot-image-yml.sh $< $@ $(shell echo ${KERNEL_TAG} | cut -d':' -f2) $(shell $(LINUXKIT) pkg show-tag pkg/xen-tools | cut -d':' -f2)
-pkg/external-boot-image: pkg/external-boot-image/build.yml
-pkg/kube: pkg/external-boot-image
-	$(MAKE) eve-external-boot-image
-	$(MAKE) cache-export IMAGE=$(shell $(LINUXKIT) pkg show-tag pkg/external-boot-image | tee pkg/kube/external-boot-image.tag) OUTFILE=pkg/kube/external-boot-image.tar && \
-	$(MAKE) eve-kube && rm -f pkg/kube/external-boot-image.tar pkg/kube/external-boot-image.tag pkg/external-boot-image/build.yml
+eve-external-boot-image: pkg/external-boot-image/build.yml
```

Comment:
Do we need this line? Doesn't [this line](https://github.com/lf-edge/eve/blob/9011acdf7d3822fe67841d7871cad21bdc4f1a80/Makefile#L1013) cover it for all `make eve-%: pkg/%`?

Reply:
Now I am confused. I see that `pkg/external-boot-image/build.yml` is in `.gitignore`. Why?

Reply:
pkg/external-boot-image/build.yml is generated at build time

Reply:
Why? Isn't that something normally checked in?

This may be irrelevant, as you move to get rid of the separate package.

Reply:
build.yml.in contains some placeholder buildArgs which are replaced when tools/compose-external-boot-image-yml.sh is run